README.md

```diff
@@ -23,19 +23,19 @@ To install this package,
 
 Examples:
 
-    openstack lease offer list
+    openstack esi offer list
```

Comment:
That's a fair question! The primary reason is to reduce the length of the command; otherwise it becomes unwieldy and somewhat irritating to type out. The other reason is that I'm thinking of this less as 'openstack / esi offer list' and more as 'openstack esi / offer list'. To use a parallel example, Ironic and Ironic Inspector are separate services in the baremetal domain; however they both use 'openstack baremetal *' CLI commands.

Let me know if that makes sense to you!

Reply:
makes sense